tools/public_api_guard/cdk/collections.d.ts

```diff
@@ -13,6 +15,11 @@ export declare abstract class DataSource<T> {
     abstract disconnect(collectionViewer: CollectionViewer): void;
 }
 
+export declare class DisposeViewRepeaterStrategy<T, R, C extends ViewRepeaterItemContext<T>> implements ViewRepeater<T, R, C> {
```

Comment:
Can we make it such that any symbols added to the public API in this change are prefixed with an underscore?

Reply:
Don't we want at least the main 3 classes to be public? `ViewRepeaterStrategy`, `DisposeViewRepeaterStrategy`, and `RecycleViewRepeaterStrategy`? Or is the idea that these will be internal and users must use a directive to provide them?

Reply:
I mainly wanted to keep them private until the virtual scroll in table is further along in case we run into something that needs to change during that work.

Reply:
Done